README.md

```diff
@@ -23,7 +24,8 @@ def largest?(array, value)
 end
 ```
 
-Snippet 2 - Big O:
+Snippet 2 - Big O: ==> This is O(2n) because when the array's size increases, the number of iterations through the array increases linearly.
+
```

Comment:
Correct reasoning, but coefficients do not matter in Big O, so we simplify to O(n)